tests/client-dbus/tests/udev/test_udev.py

```diff
@@ -424,13 +502,21 @@ def test_spurious_adds(self):
         """
         See documentation for _single_pool.
         """
-        self._single_pool(4, 4)
+        self._single_pool(4, num_hotplugs=4)
 
     def test_simple_udev_add(self):
         """
         See documentation for _single_pool.
         """
-        self._single_pool(1, 1)
+        self._single_pool(1, num_hotplugs=1)
+
+    @unittest.expectedFailure
```

Comment:
Will this line be removed after device discovery is added for encryption?

Reply:
Exactly!